waspc/data/Generator/templates/server/src/core/auth.js

```diff
@@ -69,5 +70,27 @@ export const verifyPassword = async (hashedPassword, password) => {
   }
 }
 
-export default auth
+// NOTE(shayne): This function helps us handle external auth users.
```

Comment:
"external auth users" -> I found this a bit unclear, I wasn't sure what is external auth. Maybe we could make it a bit more verbose, like this: "users authenticating via external auth provider (e.g. Google)."